frontend/src/metabase/admin/permissions/selectors.js

```diff
@@ -738,14 +738,33 @@ export const getCollectionsPermissionsGrid = createSelector(
             ];
           },
           warning(groupId, entityId) {
-            return getPermissionWarning(
-              getCollectionPermission,
-              null,
-              defaultGroup,
+            const collection = _.findWhere(collections, {
+              id: entityId.collectionId,
```

Comment:
`entityId` having a `collectionId` feels a little unexpected but not a biggie.

Reply:
That's what it's called everywhere else in the permissions code. It's an object that contains one or more properties that identifies a permissions "entity" (which predates the `metabase/entities` stuff), e.x `collectionId` or some subset of `databaseId`, `schemaName`, `tableId`.